manager/manager.go

```diff
@@ -689,6 +689,29 @@ func (m *manager) GetProcessList(containerName string, options v2.RequestOptions
 	return ps, nil
 }
 
+func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *containerData, cm collector.CollectorManager) error {
+	for k, v := range collectorConfigs {
+		configFile, err := cont.ReadFile(v, m.inHostNamespace)
+		if err != nil {
+			glog.Infof("failed to read config file %q for config %q, container %q: %v", v, k, cont.info.Name, err)
+			return err
+		}
+		glog.V(3).Infof("Got config from %q: %q", v, configFile)
+
+		newCollector, err := collector.NewCollector(k, configFile)
+		if err != nil {
+			glog.Infof("failed to create collector for container %q, config %q: %v", cont.info.Name, k, err)
+			return err
+		}
+		err = cm.RegisterCollector(newCollector)
```

Comment:
Just a small query : Can 
err = cont.cm.RegisterCollector(newCollector)

be used instead of passing cm as an argument ?